uniffi_bindgen/src/bindings/kotlin/templates/ErrorTemplate.kt

```diff
@@ -1,33 +1,9 @@
-@Structure.FieldOrder("code", "error_buf")
-internal open class RustCallStatus : Structure() {
-    @JvmField var code: Int = 0
-    @JvmField var error_buf: RustBuffer.ByValue = RustBuffer.ByValue()
-
-    fun isSuccess(): Boolean {
-        return code == 0
-    }
-
-    fun isError(): Boolean {
-        return code == 1
-    }
-
-    fun isPanic(): Boolean {
-        return code == 2
-    }
-}
-
-class InternalException(message: String) : Exception(message)
-
-// Each top-level error class has a companion object that can lift the error from the call status's rust buffer
-interface CallStatusErrorHandler<E> {
-    fun lift(error_buf: RustBuffer.ByValue): E;
-}
```

Comment:
This all disappears into `Helpers.kt`. It could go into a `ErrorRuntime.kt`, but it doesn't change and is always needed.